syft/workers/base.py

```diff
@@ -532,6 +537,33 @@ def send_command(
             responses = ret_val
         return responses
 
+    def promise_tensor(self) -> object:
+
+        ''' this works but can't pass tensor type '''
+        # pt  = sy.Promise.FloatTensor(shape=(3,3))
+        # print(pt)
+        # ref = pt.send(self)
+
+
+        '''
+        this gets error cannot serialize
+        see screen shot of stack trace:
+        https://share.getcloudapp.com/OAuLdOm6
```

Comment:
I haven't seen this particular error before, but I can say that I've been wrestling with serialization a lot, so I'm not surprised that it doesn't work as expected right off the bat. It might be useful to print and/or use the debugger to examine the contents of `simple_objects` on line 280. It should only contain Python primitive like lists, tuples, integers etc, and if there's other stuff in there that might be what's causing the issue.

Reply:
good call, I will look into that!

Reply:
I haven't seen this error either. Let's try Karl's idea.